subworkflows/local/phyloseq_workflow.nf

```diff
@@ -0,0 +1,44 @@
+/*
+ * Create phyloseq objects
+ */
+
+include { PHYLOSEQ                                } from '../../modules/local/phyloseq'
+include { PHYLOSEQ_INASV                          } from '../../modules/local/phyloseq_inasv'
+
+workflow PHYLOSEQ_WORKFLOW {
+    take:
+    ch_tax
+    ch_tsv
+    ch_meta
+    ch_tree
+    run_qiime2
+
+    main:
+    if ( params.metadata ) {
+        ch_phyloseq_inmeta = ch_meta.first() // The .first() is to make sure it's a value channel
```

Comment:
If thats not a value then the process PHYLOSEQ will run only once, correct?
An alternative might be to combine all that channels, that is how its done in other parts of the code I think, for example [here](https://github.com/nf-core/ampliseq/blob/a70ff48db034dc1fde657d426e4724793b91992a/subworkflows/local/qiime2_ancom.nf#L19-L33).
Will the value channel still cause the file to be staged properly? If not, on HPC or cloud this method might fail while it succeeds locally and on github anyway.
Also that is something that could be fixed later if it appears to be indeed a problem.

Reply:
That's correct, in my testing the process only ran once normally, but using first() to make it a value channel allowed the process to run multiple times for cases when there is multiple tax tables.

Reply:
I checked with 
```
nextflow pull a4000/ampliseq
nextflow pull a4000/ampliseq -r dev
nextflow run a4000/ampliseq -r dev -profile test,singularity --skip_qiime --sbdiexport false --outdir results_a4000_dev
```
and the work directory had soft links to 
```
ASV_table.len.tsv
ASV_tax_species.gtdb.tsv
Metadata.tsv
```
so it seems despite the first() all is properly staged. Should be fine!